src/MainPage/MainPage.js

```diff
@@ -43,9 +43,20 @@ function MainPage() {
   const [showHelp, setShowHelp] = useState(false);
   const handleHelpVisible = () => setShowHelp(!showHelp);
   const [pokemonData, setPokemonData] = useState(null);
+  const [background, setBackground] =useState();
 
-  const [playAnim, setPlayAnim] = useState(0);
-  const levelUp = () => {
+  let bgArray = [beachImg, caveImg, checkImg, cityImg, cragImg, desertImg, forestImg, savannahImg, seafloorImg, skyImg, snowImg, volcanoImg, theBeach]
+  let bgTemp = bgArray[Math.round(Math.random() * bgArray.length)];
+  
+  useEffect(() => {
+    fetchData(2);
+  }, [])
+
+  useEffect(() => {
```

Comment:
A much needed refactor, looks like the background will change upon loading, not with every use state change, which was not conducive for those sensitive to strobing lights.